libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -41,3 +41,18 @@ export const confirmedRemoveFromReadingList = createAction(
   '[Reading List API] Confirmed remove from list',
   props<{ item: ReadingListItem }>()
 );
+
+export const markBookAsFinished = createAction(
+  '[Reading List API] Mark book as finished',
+  props<{ item: ReadingListItem; finishedDate: string }>()
+);
+
+export const markBookAsFinishedSuccess = createAction(
```

Comment:
Not that important, but I would keep the naming convention of the rest of the actions (`confirmedMarkBookAsFinished` as `confirmedAddToReadingList`

Reply:
Addressed.